strawberry/aiohttp/views.py

```diff
@@ -92,18 +97,8 @@ async def handle_ws_message(
         self,
         request: web.Request,
         ws: web.WebSocketResponse,
-        ws_message: http.WSMessage,
+        message: dict,
```

Comment:
I tried, but was not very happy with the way I had to type the payload. Depending on the `type` it can basically be anything. I'll do some more experiments an let you have a look at the results.

Reply:
that looks good to me! nice one 😊